docs/user-guide.md

```diff
@@ -651,5 +756,28 @@ AI DIAL Chat supports rendering of tables and copying table data in various form
 
 ![](./img/table.png)
 
-As displayed on the illustration, you can ask a model to create a simple table. When created, you can pick one of three formats to copy its data: CSV, Markdow or Text.
+As displayed on the illustration, you can ask a model to create a simple table. When created, you can pick one of three formats to copy its data: CSV, Markdown or Text.
+
+## Administrators
+
+Administrators are chat users that can perform additional actions. Chat interface for administrators includes extra components (for example the **Approve required** tab in conversations and prompts sections)
+
+### Process Publish Requests
+
+Users with the admin role have access to the **Approve required** tab in conversations and prompts sections, where they can view all publish and unpublish requests from users. 
+
+![](./img/admin-panel.png)
+
+1. Click on any conversation or prompt request to open the review window. 
+2. In the review window, you can preview the details of the request and download attached files, if present.
+3. If **Continue review...** is displayed, click it to review conversation(s) or prompt(s) in the request. **Note**, you cannot approve the request without the review step. 
```

Comment:
Maybe we should note that "Continue to review..." only appears if some resources have been reviewed? In case no resources have been reviewed, "Go  to a review..." will be displayed.